content/tokio/topics/tracing.md

```diff
@@ -157,7 +161,7 @@ impl Handler {
         name = "Handler::run",
         skip(self),
         fields(
-            ?peer_addr = self.connection.peer_addr().unwrap(),
+            peer_addr = &self.connection.peer_addr().unwrap().to_string()[..]
```

Comment:
hmm i don't love `to_string()`ing this here...can we just record the field using `Display` instead?
```suggestion
            peer_addr = %self.connection.peer_addr().unwrap()
```

we would need to change the text on line 181 to say that it's recorded using `Display` instead of `Debug` and that `%` means `Display`, but that text was already inaccurate...